File: java/MASTER_SLAVE_WHOLE_PART_PROXY/DatosFuncion.java
import java.util.Date;
import java.util.GregorianCalendar;

public class DatosFuncion {
    //Atributos
    private String asiento;
    private Date hora;
    private String pelicula;

    //Constructor
    public DatosFuncion(){}

    //Métodos
    public void setDatosFuncion(String asiento, int dia, int mes, int anio, String pelicula){
        this.asiento = asiento;
        GregorianCalendar cal = new GregorianCalendar(anio, mes-1, dia);
        hora = cal.getTime();
        this.pelicula = pelicula;
    }

    public String getDatosFuncion(){
        return "\nDATOS DE LA FUNCION | ASIENTO: " + asiento + " | HORA: " + hora + " | PELICULA: " + pelicula;
    }
}

File: python/OBJECT_ADAPATER_CLASS_ADAPTER_WHOLE_PART/OBJECT_ADAPATER_CLASS_ADAPTER_WHOLE_PART/whole_part/DatosFuncion.py
from datetime import date 

class DatosFuncion:
    
    asiento  : str
    horario  : date
    pelicula : str
    
    def setDatosFuncion(self, asiento,
                              horario,
                              pelicula):
        self.asiento  = asiento
        self.horario  = horario
        self.pelicula = pelicula